### ISSUE TITLE
Bioportal doesn't show all the schema.org properties

### ISSUE THREAD

**Issue by johardi** [ncbo/bioportal-project#94]
The problem is because the source file of the RDF version of schema.org doesn't follow the strict rules of OWL or RDFS model.

**Comment by johardi** [comment]
Solution: https://github.com/schemaorg/schemaorg/issues/1611

tl;dr;
- replace `schema:domainIncludes` with `rdfs:domain`
- replace `schema:rangeIncludes` with `rdfs:range`
- optionally convert the `schema:DataType` statements into `rdfs:Datatype` statements

Specifically, for the third bullet point, I mapped the schema.org's datatypes to XML datatypes, added them and replaced all the literal-based range statements.
- `schema:Text`, `schema:CssSelectorType`, `schema:XPathType` mapped to `xsd:string`,
- `schema:Boolean` mapped to `xsd:boolean`
-  `schema:Date` mapped to `xsd:date`
- `schema:DateTime` mapped to `xsd:dateTime`
- `schema:Number` mapped to `xsd:decimal`
- `schema:Float` mapped to `xsd:float`
- `schema:Integer` mapped to `xsd:integer`
-  `schema:URL` mapped to `xsd:anyURI`
- `schema:Time` mapped to `xsd:time`
